resources/views/mship/waiting-lists/_waiting_lists_list.blade.php

```diff
@@ -64,6 +64,40 @@
                     @endempty
                 </tbody>
             </table>
+
+            @if ($department === \App\Models\Training\WaitingList::ATC_DEPARTMENT)
+            <div>
+                <h3>Self-enrolment</h3>
+                <p>Some waiting lists have the ability to 'self-enrol' without having to contact the Training team.
+                    If you are eligible to self-enrol, you will see a button below. Please note that this is only available for some waiting lists.</p>
```

Comment:
```suggestion
                <p>Some waiting lists have the ability to 'self-enrol' without having to contact the Training team.
                    If you are eligible to self-enrol, you will see a button below. </p>
```
Remove repeated "only some lists" as you say it at the start.